### DISCUSSION TITLE
Overflowing comment's reactions

### DISCUSSION THREAD

**Discussion by DamianGlowala** [community/community#12374] [Mobile] [answered]
![Screenshot_20220304-002630_Chrome](https://user-images.githubusercontent.com/48835293/156676117-d472766d-66b4-498c-a77c-5de81bd78556.jpg)

**Comment by stevepopovich** [comment, marked as answer]
Hi Damian, thanks for writing in. On what mobile platform are you seeing this?

**Reply by DamianGlowala** [reply]
I've just realised I opened this discussion in 'Mobile Feedback', whereas the above layout issue occured in Chrome on my mobile device (which isn't the GitHub's app). Hope you can pass it on to the relevant web team regardless :) It's a simple overflow, nothing too in-depth.

**Reply by stevepopovich** [reply]
Can do 😄

**Reply by stevepopovich** [reply]
Created https://github.com/github/feedback/discussions/12404 which _I think_ is the correct place

**Reply by DamianGlowala** [reply]
Thank you, it most likely is! 😄
